partEZ/app/Http/Controllers/Email/EmailController.php

```diff
@@ -17,10 +18,24 @@ public function sendInvitation()
         );
 
         Mail::send('emails.invitation', $data, function ($message) {
+            $message->from(env('MAIL_USERNAME'), 'partEz');
+            $message->to(Auth::user()->email)->subject('Event Invitation');
 
-            $message->from('parteznoreply@gmail.com', 'Party Planner');
+        });
+
+        return view('success');
+    }
+
+    public function sendTestEmail() 
+    {
+
+        $data = array(
+            'firstname' => Auth::user()->firstname,
+        );
 
-            $message->to('nickmanaigre@gmail.com')->subject('Event Invitation');
+        Mail::send('emails.testemail', $data, function ($message) {
```

Comment:
@delroyhiebert nope! That's just the diff display glitching out, I promise everything is properly indented! :stuck_out_tongue: